staff/sergio-arce/youtube-channel-template/index.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <div class="topbar">
+        <div>
+            <img src="https://www.youtube.com/yt/about/media/images/brand-resources/icons/YouTube_icon_dark.svg">
+        </div>
+        <div><h1 style="color:white">Boris Brejcha</h1></div>
+        <div>
+            <svg id="lupa"  viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet"><path d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"></path></svg>
+        </div>
+        <div>
+             <svg id="puntos" viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet"><path d="M12 8c1.1 0 2-.9 2-2s-.9-2-2-2-2 .9-2 2 .9 2 2 2zm0 2c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm0 6c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2z"></path></svg>
+        </div>
+    </div>
+
+
+    <div class="banner">
+        <img alt="" class="img-banner" src="//yt3.ggpht.com/JPWrmLpvj620gBgC7AB7fwMDXTQ3sJHFLXbz4BZkL_alFm_nA1tKmUzsSpY0DsnyMNbH58kkMew=w2560-fcrop64=1,00005a57ffffa5a8-nd-c0xffffffff-rj-k-no">
+    </div>
+
+    <div class="profile">
+        <div>
+            <img class="img-profile" alt="" src="https://yt3.ggpht.com/a-/AAuE7mDLkzUi01fI4IREQHczEXo7bEReeX5GLXARBg=s288-mo-c-c0xffffffff-rj-k-no">
+        </div>
+        <div>Boris Brejcha</div>
+        <div>
+            imagen
+        </div>
+        <div>numero suscriptores</div>
+    </div>
+
+    <div class="navbar">
+        <div>inicio</div>
+        <div>videos</div>
+        <div>listas</div>
+        <div>
+            <svg id="log-1" viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet"><path d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"></path></svg>
+        </div>
+    </div>
+
+    <div class="trailer">
+        <div>video</div>
+        <div>titulo video</div>
+    </div>
+
+    <div class="galeria">
```

Comment:
@sergi23 with BEM, this could be:
`album-container`, `album-container__album`, `album-container__album-video`, `album-container__album-title`, `album-container__album-author`, `album-container__album-view`, `album-container__album-menu`

Reply:
thanks, done